src/bit_vector.rs

```diff
@@ -168,6 +168,10 @@ impl FID for BitVector {
         self.len
     }
 
+    fn is_empty(&self) -> bool {
```

Comment:
Could you move this implementation to `FID` trait as a default method?

Reply:
Done in commit #02e5742.